pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1061,6 +1061,19 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 					})
 				}
 			}
+			// verify that the extra dhcp options are valid
+			if iface.DHCPOptions != nil {
+				for _, index := range iface.DHCPOptions.ExtraOptions {
+					if !(index.Option >= 224 && index.Option <= 254) {
```

Comment:
You probably also want to check for duplicates in the list.

Reply:
@rmohr, Done, added the validation and also more tests!